File: Consecutive_Characters.java
class Solution {
    public int maxPower(String s) {
        char maxChar;
        int max = 0;
        char curChar = s.charAt(0);
        int cur = 1;
        for (int i = 1; i < s.length(); i++) {
            if (s.charAt(i) == curChar) {
                cur++;
            } else {
                if (cur > max) {
                    maxChar = curChar;
                    max = cur;
                }
                curChar = s.charAt(i);
                cur = 1;
            }
        }
        return Math.max(max, cur);
    }
}

File: Consecutive_Characters.cpp
class Solution {
public:
    int maxPower(string s) {
        int count=0;
        int Max=0;
        for(int i=0;i<s.length();i++){
            if(s[i]==s[i+1]){
                count++;
            }
            Max=max(Max,count);
            if(s[i]!=s[i+1]){
                count=0;
            }
        }
        return Max+1;
    }
};
